question/bank/viewcreator/classes/columns.php

```diff
@@ -0,0 +1,53 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Plugin entrypoint for columns.
+ *
+ * @package    qbank_viewcreator
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Safat Shahin <safatshahin@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace qbank_viewcreator;
+
+defined('MOODLE_INTERNAL') || die();
+
+use core_question\local\bank\column_plugin_base;
+
+/**
+ * Class columns is the entrypoint for the columns.
+ *
+ * @package    qbank_viewcreator
+ * @copyright  2021 Catalyst IT Canada Pty Ltd
```

Comment:
Please use Catalyst IT Australia in all copyright as matt recommended

Reply:
Hi @safatshahin :

- Copyright in `version.php`, `columns.php` and `provider.php` set to Australia
- Phpunit tests ok
- Codechecker run and ok